dev_environment/setup_dev_env_linux_vm.md

```diff
@@ -0,0 +1,26 @@
+# Set up L3AF Development Environment for Linux VMs
+
+There are different ways to create a Linux (i.e., Ubuntu 20.04) VM.  On a local computer host, various VM software, e.g., VirtualBox, Parallels, Multipass, Windows WSL, etc. can create a Linux VM.  A Linux VM can also be created in a cloud environment, e.g., Azure VM.  The following procedures illustrate how to set up a L3AF development environment for those VMs if Vagrant doesn't support them.
+
+# Host Prerequisites
+
+* [L3AFD source code](https://github.com/l3af-project/l3afd)
+* [curl](https://curl.se/)
+* [hey](https://github.com/rakyll/hey) or any HTTP load generator
+* A web browser
+* A Linux VM running on the host or in a cloud (record the login information while creating the Linux VM, and root access is required.)
+
+# Trying out L3AF
+
+1. Edit `config.yaml` to point to the [source code](https://github.com/l3af-project/l3afd) on your host machine. This
```

Comment:
Is this needed as we download it directly to the VM in the script below?

Reply:
Yes, we needed here.  So after the source code got downloaded and map into the VM, the config.yaml pointed to the right location.

Reply:
I don't see your script referencing config.yaml, so not sure why this is needed outside vagrant

Reply:
The code path is in the root, so we don't need to point to some developers own directory.  I took out the step 1.